DuckDuckGo/AppDelegate.swift

```diff
@@ -117,6 +124,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         if !privacyStore.authenticationEnabled {
             showKeyboardOnLaunch()
         }
+
+        AppConfigurationFetch().start { newData in
+            if newData {
+                NotificationCenter.default.post(name: ContentBlockerProtectionChangedNotification.name, object: nil)
```

Comment:
Open to suggestions for a better way to react to tracker data changes – this is just a way to have `TabViewController.onContentBlockerConfigurationChanged` called since we want rules recompiled, but I'm always skeptical about falling back on notifications since they can make code harder to follow.

(Am also open to defining a separate notification for this, since the name of this one isn't perfectly suitable.)

Reply:
This seems fine to me, notifications should be a valid pattern to use here as there may be multiple observers and at least few places possibly triggering this to fire.